Cylenium/cy.cs

```diff
@@ -245,6 +245,49 @@ public static void Screenshot(string fileName)
 
         #region BROWSER
 
+        /// <summary>
+        /// Adds a cookie to the current page.
+        /// </summary>
+        /// <returns>The cookie that was added.</returns>
+        public static Cookie AddCookie(Cookie cookie)
+        {
+            WebDriver.Manage().Cookies.AddCookie(cookie);
```

Comment:
so I see you have 2 different ways that you do return, the one liner with GetCookie and 2 liner with AddCookie. Is there an advantage one way or the other?

Reply:
In this specific scenario, there is no advantage. `AddCookie()` is void, meaning it doesn't return anything. So all I'm doing is returning the cookie you passed in.

`GetCookie()` does return something which is why I'm just returning whatever that returns